js/vue/meetupEvents.vue

```diff
@@ -0,0 +1,96 @@
+
+<template>
+  <div>
+    <div class="meetups">
+      <div class="meetup-event" v-for="event in events" :key="event.id">
+        <h3 class="meetup-heading"> {{ event.name }}</h3>
+        <hr>
+        <div class="meetup-description">
+          <div v-html="event.description"></div>
+          <div class="meetup-details">
+            <div class="meetup-date"> <b>When:</b> {{ event.parsed_date }} </div>
+            <div v-if="event.venue" class="meetup-location"> <b>Where:</b> {{ event.venue.name }} </div>
+            <div v-else> <b>Where:</b> TBD </div>
+            <div class="meetup-who"> <b>Who:</b> {{ event.yes_rsvp_count }} already going</div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+//import node module dependencies
+import jsonp from 'jsonp';
+
+//import utility dependencies
+import { parameterize, formatDate } from '../utils/component-utils.js';
+
+
+const DESIRED_HACK_NIGHTS = 4;
+const HACK_NIGHT = "[Everyone] Full Stack Hack Night";
+const EVENT_URL = "http://api.meetup.com/newhavenio/events?";
```

Comment:
The protocol should be removed to prevent possible insecure AJAX errors.

```js
const EVENT_URL = "//api.meetup.com/newhavenio/events?";
```

Reply:
noted, i haven't done a ton of requests so i didn't know you could do that. Learn something new every day